lib/GithubUtils.js

```diff
@@ -0,0 +1,76 @@
+export const GithubOwner = 'Expensify';
+export const ExpensifyIssueRepo = 'Expensify';
+export const ExpensifyCashRepo = 'Expensify.cash';
+
+export default class GithubUtils {
+    /**
+     * @param {Octokit} octokit - Authenticated Octokit object https://octokit.github.io/rest.js
+     */
+    constructor(octokit) {
+        this.octokit = octokit;
+    }
+
+    /**
+     * Finds one open `StagingDeployCash` issue via GitHub octokit library
+     * @returns {Promise}
+     */
+    getStagingDeployCash() {
+        return this.octokit.issues.listForRepo({
+            owner: GithubOwner,
+            repo: ExpensifyIssueRepo,
+            labels: 'StagingDeployCash',
+            state: 'open'
+        })
+            .then(({data}) => {
+                if (!data.length) {
+                    throw new Error('Unable to find StagingDeployCash issue.');
+                }
+
+                if (data.length > 1) {
+                    throw new Error('Found more than one StagingDeployCash issue.');
+                }
+
+                return this.getStagingDeployCashData(data[0]);
+            });
+    }
+
+    /**
+     * Takes in a GitHub issue object and returns the data we want
+     *
+     * @param {Object} issue
+     * @returns {Promise}
+     */
+    getStagingDeployCashData(issue) {
+        try {
+            const versionRegex = /`([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9]*))?`/g;
+            const tag = issue.body.match(versionRegex)[0].replace(/`/g, '');
+
+            // eslint-disable-next-line max-len
+            const compareURLRegex = /https:\/\/github\.com\/Expensify\/Expensify.cash\/compare\/([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9]*))?...([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9]*))?/g;
+            const comparisonURL = issue.body.match(compareURLRegex)[0];
+
+            const PRListRegex = /https:\/\/github\.com\/Expensify\/Expensify.cash\/pull\/([0-9]+)/g;
+            const PRList = issue.body.match(PRListRegex);
+
+            const data = {
+                title: issue.title, url: issue.url, labels: issue.labels, tag, comparisonURL, PRList
+            };
+
+            return this.octokit.git.getRef({
```

Comment:
Hmmm I'm not sure this will work going forward to determine if the `StagingDeployCash` is deployed to production. But also, I'm not sure if we'll even need `isDeployedToProdction`, since in general any open `StagingDeployCash` _should_ be deployed to staging and any closed `StagingDeployCash` _should_ be deployed to production. The way to tell for sure would be to check if there is a release matching the tag you are looking for.

Reply:
Ok I am happy to either remove it or use `octokit.repos.getReleaseByTag`, just let me know which you'd prefer.

Reply:
I looked through the design doc and it looks like we're not using `isDeployedToProduction` anywhere, so we should just remove it.